spring-cloud-gcp-samples/pom.xml

```diff
@@ -43,6 +43,18 @@
 		<module>spring-cloud-gcp-data-multi-sample</module>
 	</modules>
 
+	<dependencyManagement>
+		<dependencies>
+			<dependency>
+				<groupId>org.springframework.cloud</groupId>
+				<artifactId>spring-cloud-gcp-dependencies</artifactId>
+				<version>1.2.0.BUILD-SNAPSHOT</version>
+				<type>pom</type>
+				<scope>import</scope>
+			</dependency>
+		</dependencies>
+	</dependencyManagement>
+
 	<build>
 		<plugins>
 			<plugin>
```

Comment:
I don't think you need this plugin here `spring-boot-maven-plugin`.

Reply:
Done.